packages/js/product-editor/src/components/header/style.scss

```diff
@@ -23,13 +23,26 @@
 		overflow: hidden;
 		text-overflow: ellipsis;
 
-		@include breakpoint( '<960px' ) {
+		@include breakpoint("<960px") {
```

Comment:
Nitpicky, maybe we could use `@include break-large()` instead of this. It should [be the same](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss#L10).

Reply:
Yep, it looks an auto format since this line is not part of the PR